components/focus-trap/focus-trap.js

```diff
@@ -34,6 +34,15 @@ class FocusTrap extends FocusMixin(LitElement) {
 		`;
 	}
 
+	static #exemptSelectors = [
```

Comment:
Is it worth adding a built-in one like `.d2l-focus-trap-exempt` that Equatio (or some other future thing) could possibly put on their stuff, allowing us to not have references to their classes in our code?